ImageViewer/Source/ItemBaseController.swift

```diff
@@ -159,6 +159,7 @@ open class ItemBaseController<T: UIView>: UIViewController, ItemController, UIGe
             swipeToDismissRecognizer.addTarget(self, action: #selector(scrollViewDidSwipeToDismiss))
             swipeToDismissRecognizer.delegate = self
             view.addGestureRecognizer(swipeToDismissRecognizer)
+            swipeToDismissRecognizer.require(toFail: doubleTapRecognizer)
```

Comment:
Can you explain in the PR description why this change is necessary?  There's no mention of it.